install/installer/tools/ambari_setup

```diff
@@ -62,6 +65,9 @@ done
 #Info for log file
 sudo mkdir -p /var/log/trafodion
 sudo chmod 777 /var/log/trafodion
+
+linuxInstalled=$(grep -cr SUSE /proc/version)
```

Comment:
Actually the grep -c returns a number value. So if SUSE is not installed, it will return a 0 not an empty string. I had to go test this out to make sure if does that (as I couldn't recall) and it does. I did change the variable name to be more clear/.